settings.lua

```diff
@@ -199,6 +199,9 @@ end;
 function courseplay:setDriveUnloadNow(vehicle, bool)
 	if vehicle then
 		vehicle.cp.settings.driveUnloadNow:set(bool)
+		if bool then 
+			vehicle.cp.driver.triggerHandler:setDriveNow()
```

Comment:
Again, not sure about the driveNow handling, but the knowledge about when to call the triggerHandle:setDriveNow() should be internal to AIDriver (there's already an AIDriver:setDriveUnloadNow()

Reply:
triggerHandle:setDriveNow()  gets now called by the button directly in courseplay:setDriveNow()